README.md

```diff
@@ -94,4 +94,17 @@ $ mvn archetype:generate -B \
      -DprojectName=<project-name>  \
      -DcomponentGroup=<component-group> \
      -DoptionFrontend=react
-```
\ No newline at end of file
+```
+
+## Guidelines for SPA development in AEM
+
+Developing single page applications on AEM assumes that the front-end developer observes standard best practices when creating an SPA. If as a front end developer you follow these general best practices as well as few AEM-specific principles, your SPA will be functional with AEM and its content-authoring capabilities.
+
+* *Portability* - As with any components, the components should be built to be as portable as possible. The SPA should be built with portably and reusable components.
```

Comment:
`portable and reusable components`?

Reply:
Here we could add that, in particular, one should avoid using static paths that refer to the content structure as paths can be modified at any time by the content authors. It would also restrict the reusability of the library. And it prevents the template editor to be used (as its structure is located in another location than the content)